DigitalLearningSolutions.Web.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -1629,19 +1722,19 @@ private void GivenAdminsToNotifyHaveEmails(IEnumerable<string> adminEmails)
                 );
         }
 
-        private void RegistrationRequiresApprovalEmailMustHaveBeenSentTo(NotificationRecipient notificationRecipient)
+        private void RegistrationRequiresApprovalEmailMustHaveBeenSentTo(string recipientEmail, int numberOfTimes = 1)

```

Comment:
Noticed that this method was unused, so I added it in the tests above where it was intended to go

Reply:
Is it still unused?

Reply:
No, it's now used in the tests above